src/modules/popup.js

```diff
@@ -1,19 +1,64 @@
-const displayPopup = (meal) => {
+import { saveComment, handleSaveComment } from './saveComment.js';
+
+const displayPopup = (meal, appId) => {
+  const popupOverlay = document.querySelector('.popup-overlay');
+
   const popup = document.querySelector('.popup');
-  popup.style.display = 'block';
+  popup.style.display = 'flex';
+  popupOverlay.style.display = 'flex';
   popup.innerHTML = `
     <i class="fa fa-times close" aria-hidden="true"></i>
     <img class="popup-img" src="${meal.strMealThumb}" alt="">
     <h1 class="food-name">${meal.strMeal}</h1>
     <ul class="food-info">
     <li>Area : ${meal.strArea}</li>
     <li>Tags : ${meal.strTags}</li>
```

Comment:
I suggest that adding some CSS in your popup will improve your project and make it look alike to the wireframe. The image must dominate and not the title.
![Screenshot (27)](https://github.com/Bhekisile/Capstone_JavaScript/assets/111288560/7cd8e8e3-ea47-43c5-93e7-37bd0daa7361)
![Screenshot (25)](https://github.com/Bhekisile/Capstone_JavaScript/assets/111288560/3a171a78-b312-42e8-bc8b-21fdd2242c5e)